File: Service/Front/pet_tale/src/api/urls.js
export const HOST = 'http://127.0.0.1:8000';

export const urls = {
    base: HOST,
    itemList : (params) => {
        let url = new URL(`${HOST}/api/v0/itemlist`);
        params.forEach((param) => {url.searchParams.set(param[0], param[1]);})
        return url;
    },
    item : (itemID) => `${HOST}/api/v0/item/${itemID}`,

    login : (login, psw) => `${HOST}/api/v0/login?login=${login}&psw=${psw}`,
    register : (login, email, psw) => `${HOST}/api/v0/register?login=${login}&email=${email}&psw=${psw}`,

    buy : (login, item_id) => `${HOST}/api/v0/buy?login=${login}&item=${item_id}`,

    getSimilar : (item_id) => `${HOST}/api/v0/fitfit/similar?item=${item_id}`,
    getRecoms : (item_id, user_id) => `${HOST}/api/v0/fitfit/recommendations?item=${item_id}&user=${user_id}`,
}

File: Service/Server/PetTale/api_v0/urls.py
from django.conf.urls import url
from api_v0 import views


urlpatterns = [
    url(r'^itemlist$', views.goodList, name='goodList'),
    url(r'^item/(\d+)$', views.goodItem, name='goodItem'),

    url(r'^login$', views.login, name='login'),
    url(r'^register$', views.register, name='register'),

    url(r'^buy$', views.buy, name='buy'),

    url(r'^fitfit/similar$', views.similar, name='similar'),
    url(r'^fitfit/recommendations$', views.recommendations, name='recommendations'),
    #
]